toolkit/tools/internal/rpm/rpm.go

```diff
@@ -89,8 +91,19 @@ var (
 	//
 	//	D: ========== +++ systemd-devel-239-42.azl3 x86_64-linux 0x0
 	installedRPMRegex = regexp.MustCompile(`^D: =+ \+{3} (\S+) (\S+)-linux.*$`)
+
+	// For most use-cases, the distro name abbreviation and major version are set by the exe package. However, if the
+	// module is used outside of the main Mariner build system, the caller can override these values with SetDistroMacros().
+	distNameAbreviation = exe.DistroNameAbreviation
+	distMajorVersion    = exe.DistroMajorVersion
```

Comment:
We could now also move the error checks inside `DefaultDistroDefines` to here, so that they run when the application gets started. This way we'd fail faster for invalid values and avoid having to push the `error` through the code.

Something along these lines:
```suggestion
	distNameAbreviation = exe.DistroNameAbreviation
	distMajorVersion    = ldFlagToPositiveInt(exe.DistroMajorVersion, "DistroMajorVersion")
	
	func ldFlagToPositiveInt(name, value string) int {
	result, err := strconv.Atoi(name)
	if err != nil || result <= 0 {
		err = fmt.Errorf("failed to parse Go ldflag '%s'='%s'. The value must be a positive integer", name, value)
		panic(err)
	}

	return result
}
```